configure

```diff
@@ -130,6 +130,8 @@ DSC_SCRIPT_PATH=$DSC_SCRIPT_PATH
 
 BUILD_DEBUG=$BUILD_DEBUG
 DEBUG_FLAGS=$DEBUG_FLAGS
+# DSC cannot build with -Werror
+ENABLE_WERROR=0
```

Comment:
I think it's because `$(TOP)/config.mak` gets included (again) in later Makefiles, bringing in this override, see [here](https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/develop/LCM/dsc/engine/lcm/GNUmakefile#L3) (note that the include at the bottom is *not* OMI's `config.mak`, it's the `rules.mak` which does not set `-Werror`).